src/components/common/AudioLibrarySearchField.vue

```diff
@@ -69,102 +70,102 @@
 </template>
 
 <script>
-	import fuse from 'fuse.js'
-	import lodash from 'lodash'
-	import VueFocus from 'vue-focus'
-  import lang from '@/lib/filters/lang'
-
-	export default {
-		props: ['langId', 'audioFiles'],
-		mixins: [VueFocus.mixin, lang],
-
-		data() {
-			return {
-				isActive: false,
-
-        // querying
-				rawQuery: '',
-				cache: {},
-        isEntireLibraryModeEnabled: false,
-
-        // pagination
-        offset: 0,
-        limit: 10,
-			}
-		},
-
-		computed: {
-      query() {
-				return lodash.trim(this.rawQuery)
-      },
-
-      isAudioLibraryEmpty() {
-				return this.isActive && !this.audioFiles.length
-      },
-
-      hasNext() {
-        return (this.search(this.query).length / (this.offset + 1)) > this.limit
-      },
-
-      hasPrevious() {
-        return this.offset > 0
-      },
-		},
-
-		methods: {
-			search(query) {
+import fuse from 'fuse.js';
```

Comment:
@bzabos , can you remind us our rule for semi-colon ?
If we expect trailing semi, shall we add a rule in `eslintrc` to avoid this kind of change ?

Reply:
Good one, I've been pushing for us to skip semi-colons!